### PR TITLE
Add SetUTF8()

### PR DESCRIPTION
**Pull request by zeast**
I need command `opts utf8 on` to display Chinese from FTP server

### CODE REVIEW

**Comment by corny** [comment]
Could it be easier to always send a `opts utf8 on` when the server offers `UTF8` in the  `FEAT` response? There is also a test missing. Why did you close this pull request?

**Comment by zeast** [comment]
@corny 
Q: Could it be easier to always send a opts utf8 on when the server offers UTF8 in the FEAT response?
A: I think it's work for me. But always use utf8 on is a better way? I worried that others don't want to use this option.

Q: There is also a test missing
A: I realize that. I thought the coverage is auto. Now, I know I must write a test, coverage results just like go testflag -cover.

Q: Why did you close this pull request?
A: When I realize test missing, I want to set pull request status close and write a test and pull request again. This is my first pull request so maybe I do the wrong thing.

File: ftp.go [hunk 1]
```diff
@@ -687,6 +687,32 @@ func (c *ServerConn) Quit() error {
...
+	}
+
+	switch code {
+
+	case StatusCommandOK:
+		return nil
+
+	default:
+		return errors.New(message)
+	}
```
Comment (corny) [inline ftp.go]: I prefer:
```go
	if code != StatusCommandOK {
		return errors.New(message)
	}

	return nil
```

**Comment by corny** [comment]
In my opinion UTF8 should be the default. If it's not UTF8, what is it then? :)

File: ftp.go (hunk 1 above)
Comment (zeast) [inline ftp.go]: nice.  And you remind me, the function need check `ServerConn.features`  first, not all server offers UTF8 option!

**Comment by zeast** [comment]
@corny  I agree with you.

**Pull request closed without merging**